extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -683,6 +683,10 @@ export class Bind {
   applyBinding_(boundProperty, element, newValue) {
     const property = boundProperty.property;
     switch (property) {
+      case 'value':
```

Comment:
It sets the attribute, but that only works the first time. Subsequent changes to the value attribute don't change the text inside the element : /